wnt/protocol_buffers_files/message.proto

```diff
@@ -197,7 +198,8 @@ message DiagnosticsData {
     optional float cb_mac_load = 51;
 
     // If WM version >= 4.0 Access cycles from traffic diagnostics
-    // is replaced by cluster node amounts
+    // is replaced by cluster node amounts.
+    // Back in WM FW 5.2
```

Comment:
Just to confirm, this field was not "active in previous versions", but is "returned" to the message in 5.2? Could the wording be something like this:

`// was replaced by cluster_xxx_node_amounts in previous versions, but is activated again in WM firmware version 5.2`